include/zephyr/bluetooth/bluetooth.h

```diff
@@ -176,13 +224,13 @@ struct bt_le_ext_adv_cb {
 	/**
 	 * @brief The RPA validity of the advertising set has expired.
 	 *
-	 * This callback notifies the application that the RPA validity of
-	 * the advertising set has expired. The user can use this callback
-	 * to synchronize the advertising payload update with the RPA rotation.
+	 * This callback notifies the application that the RPA validity of the advertising set has
+	 * expired. The user can use this callback to synchronize the advertising payload update
+	 * with the RPA rotation by for example envoking @ref bt_le_ext_adv_set_data upon callback.
 	 *
-	 * If rpa sharing is enabled and rpa expired cb of any adv-sets belonging
-	 * to same adv id returns false, then adv-sets will continue with old rpa
-	 * through out the rpa rotations.
+	 * If rpa sharing is enabled (see @kconfig{CONFIG_BT_RPA_SHARING}) and the rpa expired cb of
```

Comment:
```suggestion
	 * If RPA sharing is enabled (see @kconfig{CONFIG_BT_RPA_SHARING}) and the RPA expired callback of
```

Consider adding a reference to the mentioned callback